client/components/popover/style.scss

```diff
@@ -201,7 +201,7 @@
 	}
 
 	// Menu Items with Icons
-	&.has-icon {
+	&.has-icon:not( .external-link ) {
```

Comment:
I'm finding this a bit difficult to decipher.  Why do we not need this padding for external links?  Is it worth adding a comment about this situation?

Reply:
@nylen: Good question. I tried looking through the popover menu history, but I actually think `.has-icon` is a legacy class that no longer gets applied. All of the icons get rendered inline, thus the padding doesn't make sense. In this particular case, the `ExternalLink` component is adding the `.has-icon` class, which is being styled upstream. I'd like to remove the style completely, but was afraid of regressions.